README_GITFLEXIMOD.rst

```diff
@@ -18,33 +13,33 @@ To obtain the CTSM code you need to do the following:
 
    This will create a directory ``my_ctsm_sandbox/`` in your current working directory.
 
-#. Run the script **manage_externals/checkout_externals**. ::
+#. Run **./bin/git-fleximod update**. ::
 
-      ./manage_externals/checkout_externals
+      ./bin/git-fleximod update
 
-   The **checkout_externals** script is a package manager that will
+   **git-fleximod** is a package manager that will
    populate the ctsm directory with the relevant versions of each of the
    components along with the CIME infrastructure code.
 
 At this point you have a working version of CTSM.
 
 To see full details of how to set up a case, compile and run, see the CIME documentation at http://esmci.github.io/cime/ .
 
-More details on checkout_externals
-----------------------------------
+More details on git-fleximod
+----------------------------
 
 The file **Externals.cfg** in your top-level CTSM directory tells
```

Comment:
The Externals.cfg file no longer exists, the correct file is .gitmodules.  See https://github.com/ESMCI/git-fleximod?tab=readme-ov-file#git-fleximod for documentation of git-fleximod.